src/_scss/pages/awardV2/idv/_awardFundingSummary.scss

```diff
@@ -1,5 +1,6 @@
 .federal-accounts {
   width: 100%;
+  @import 'components/Note';
```

Comment:
Not sure I agree with this pattern in our code base... Couldn't we just import all the partials in a parent-partial? It seems to me we're importing the same css multiple times, when we could just import it once. This would appear to remove duplicate CSS.

Webpack removes unused code by default, but I think this is one of the reasons our build takes a long time.

Perhaps I'm wrong about this but I'd like to have a discussion about it!